.github/workflows/label_when_approved.yml

```
@@ -0,0 +1,65 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+---
+name: Label when approved
+on: pull_request_review  # yamllint disable-line rule:truthy
+
+jobs:
+
+  label-when-approved:
+    name: "Label PRs when approved"
+    runs-on: ubuntu-latest
+    outputs:
+      isApprovedByCommiters: ${{ steps.label-when-approved-by-commiters.outputs.isApproved }}
+      isApprovedByAnyone: ${{ steps.label-when-approved-by-anyone.outputs.isApproved }}
+    steps:
+      - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
+        uses: actions/checkout@v2
+      - name: >
+          Event: ${{ github.event_name }}
+          Repo: ${{ github.repository }}
+          Branch: ${{ github.head_ref }}
+          Run id: ${{ github.run_id }}
+          Sha: ${{ github.sha }}
+          Ref: ${{ github.ref }}
+        run: printenv
+      - name: Selective checks
+        id: selective-checks
+        env:
+          EVENT_NAME: ${{ github.event_name }}
+          INCOMING_COMMIT_SHA: ${{ github.sha }}
+          PR_LABELS: "${{ steps.source-run-info.outputs.pullRequestLabels }}"
+        run: |
+          ./scripts/ci/selective_ci_checks.sh "${INCOMING_COMMIT_SHA}"
+      - name: Label when approved by commiters for PRs requiring full tests
+        uses: TobKed/label-when-approved-action@master
+        id: label-test-prs-when-approved-by-commiters
+        if: steps.selective-checks.outputs.run-tests
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          label: 'okay to test'
+          require_committers_approval: 'true'
+          # TODO: add comments and checks
+      - name: Label when approved by commiters for PRs that do not require tests
+        uses: TobKed/label-when-approved-action@master
+        id: label-non-test-prs-when-approved-by-commiters
+        if: ! steps.selective-checks.outputs.run-tests
```

```suggestion
        if: !steps.selective-checks.outputs.run-tests
```

---

maybe this, I am not sure
```suggestion
        if: steps.selective-checks.outputs.run-tests != 'true'
```

---

Yep. I corrected it already. Can you disapprove/approve again please :)